### PR TITLE
Whats on - event listing and featured card styles

### PR DESCRIPTION
**Pull request by helenb**
## About these changes

Styling for the whats on page event listing and featured card
 
    - Hard-codes the values for the featured card for now
    - Includes a few tweaks to the filters and event card styles - some of these need porting back to my storybook pr
    - Adds a new component for the featured card, as the layout was sufficiently different from the base card
    - Moves the meta styles into their own component as they are now shared between the featured card and the card

## How to check these changes

 - On your local build, check out the feature/whats-on-listing-fe branch
 - Create a 'whats on listing page' and add some event pages below it

## Screenshots

<details>
<summary>Click to view screenshots</summary>

Desktop
screencapture-localhost-8000-whats-on-2023-10-06-12_37_28

Mobile
screencapture-localhost-8000-whats-on-2023-10-06-12_37_55

</details>

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer
- [ ] Checked PR title starts with ticket number as per project conventions to help us keep track of changes
- [x] Ensured that PR includes only commits relevant to the ticket
- [x] Waited for all CI jobs to pass before requesting a review
- [ ] Added/updated tests and documentation where relevant

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% load static wagtailuserbar wagtailcore_tags wagtailsettings_tags  wagtailmetadata_tags robots_meta %}<!DOCTYPE html>
 {% get_settings %}
-<html class="tna-template tna-template--light-theme tna-template--yellow-accent no-js" lang="en-GB">
+<html class="tna-template tna-template--light-theme tna-template--yellow-accent no-js {% block html_class%}{% endblock %}" lang="en-GB">
```
Comment (helenb) [inline templates/base.html:3]: New block to allow overriding of the accent colour per page

File: templates/components/card.html [hunk 2]
```diff
@@ -52,7 +52,7 @@
     {% endif %}
     {% if imageSrc %}
       {% if href %}
-      <a href="{{ href }}" class="tna-card__image-container" tabindex="-1">
+      <a href="{{ href }}" class="tna-card__image-container" tabindex="-1" title="{{ title }}">
```
Comment (helenb) [inline templates/components/card.html:55]: I added in a title here as axe complained about the lack of discernible text on the link

**Comment by helenb** [comment]
@ahosgood for your reference.

**Comment by LeanneEmans** [comment]
@ahosgood Hi AJ please could you give this PR a review?

**Review by ahosgood — APPROVED** [review]

**Comment by LeanneEmans** [comment]
@helenb just to let you know, @ahosgood AJ has approved this PR
So when you are next back to work with us (Monday 16th) please kindly merge this branch
Thank you!

**Pull request merged**